imports/plugins/core/dashboard/register.js

```diff
@@ -50,7 +50,7 @@ Reaction.registerPackage({
     enabled: true,
     structure: {
       template: "dashboardPackages",
-      layoutHeader: "layoutHeader",
+      layoutHeader: "NavBar",
```

Comment:
Any time these values change, you'll need to write a migration. This goes for all the `register.js` files modified.

Migrations can be found in `/imports/plugins/core/versions`.

Reply:
thanks. I've written a migration for it.